### PR TITLE
open/gui: when loading a failed global route, advertise congestion report

### PR DESCRIPTION
**Pull request by oharboe**
Instead of giving an error.

Single source of truth as to the name of the global routing congestion report.

### CODE REVIEW

File: flow/scripts/open.tcl [hunk 1]
```diff
@@ -41,7 +41,11 @@ proc read_timing {input_file} {
   if {$design_stage >= 6 && [file exist $::env(RESULTS_DIR)/6_final.spef]} {
     log_cmd read_spef $::env(RESULTS_DIR)/6_final.spef
   } elseif {$design_stage >= 5} {
-    log_cmd estimate_parasitics -global_routing
+    if { [grt::have_routes] } {
+      log_cmd estimate_parasitics -global_routing
+    } else {
+      puts "No global routing results: load $::global_route_congestion_report for details"
```
Comment (maliberty) [inline flow/scripts/open.tcl]: If GUI=1 then give a warning about missing parasitics for timing.

Comment (oharboe) [reply flow/scripts/open.tcl]: Added warning, but why should GUI=1/0 be different in open/gui?

Shouldn't the only difference between open/gui_ be console vs. GUI?

Comment (maliberty) [reply flow/scripts/open.tcl]: Sorry I meant GUI_TIMING=1

Comment (oharboe) [reply flow/scripts/open.tcl]: Yes, GUI_TIMING=1/0 should have the same behavior. GUI vs. console; same behavior here.

**Review by maliberty — APPROVED** [review]

**Pull request merged**